source/lib/cdk-textract-stack.ts

```diff
@@ -73,14 +74,27 @@ export class CdkTextractStack extends cdk.Stack {
     };
 
     // S3 buckets
+    const logsS3Bucket = new s3.Bucket(
+      this,
+      this.resourceName("LogsS3Bucket"),
+      {
+        bucketName: this.resourceName("logs-s3-bucket"),
```

Comment:
NIT: can we call this s3-access-logs-bucket?
I want to avoid any situation where the customer might confuse this to have all logs for the solution

Reply:
Resolving this since this bucket is also being used for Cfn Distro logs